INSTALL.txt

```diff
@@ -2,20 +2,94 @@ Installers for Windows and Mac OS X and tarballs for Linux can be found at:
 
     http://sourceforge.net/projects/unimediaserver/files/Official%20Releases/
 
-To install UMS from the tarball on Linux, open a terminal and enter the
-following commands (replace <version> with the version you're installing):
-Note: These instructions have been tested on Ubuntu 12.04, but something similar
-should work on most Unix distributions)
+To install UMS from the tarball on Linux, open a terminal and enter the following commands (replace <version> with the version
+you're installing):
+Note: These instructions have been tested on Ubuntu 12.04, but something similar should work on most Unix distributions)
 
 1) Install the dependencies (this only needs to be done once):
 
-    sudo apt-get install mediainfo openjdk-7-jre
-    # you can also optionally install dcraw and VLC: sudo apt-get install dcraw vlc
+    # UMS requires a Java Virtual Machine to run. Which version depends on which binary package was downloaded. UMS for Java 7
+    would need version 7 or newer, UMS for Java 8 would need version 8 or newer. The most common JVM's to use on Linux is
+    openjdk. Which, if any, of the packages that's available depends on your distro, it's version and what sources are enabled.
+
+    # Install JVM on Debian/Ubuntu and derivates:
+      Debian 8 need to use backports to install openjdk-8 at the time of writing. It's not possible to cover all the sources
+      needed for the different versions in this document. If the package isn't found, make sure to run `apt-get update` to
+      make sure the sources are refreshed.
+      # To install openjdk-7 run:
+      sudo apt-get install openjdk-7-jre
+      # To install openjdk-8 run:
+      sudo apt-get install openjdk-8-jre
+
+    # Install JVM on Fedora, OpenSUSE, Mandriva, Red Hat/Centos and derivates: The epel repository might be needed on some
+    platforms.
+      # To install openjdk-7 run:
+      sudo yum install java-1.7.0-openjdk
+      # To install openjdk-8 run:
+      sudo yum install java-1.8.0-openjdk
+
+    # Install JVM on Arch:
+      # To install openjdk-7 run:
+      sudo pacman -S jre7-openjdk
+      # To install openjdk-8 run:
+      sudo pacman -S jre8-openjdk
+
+    # UMS use MediaInfo to get information about media files. It can be run without it, but media handling will suffer as UMS
+    won't have a detailed information available.
+    # Install MediaInfo on Debian/Ubuntu and derivates:
+    sudo apt-get install mediainfo
+    # Install MediaInfo on Fedora, OpenSUSE, Mandriva, Red Hat/Centos and derivates:
+    sudo yum install mediainfo
+    # Install MediaInfo on Arch (community repository must be enabled):
+    sudo pacman -S mediainfo
+
+    # You can optionally install dcraw (to decode raw images from digital cameras) and VLC (to use as a transcoding engine).
```

Comment:
It could be good to split the optional dependencies into an "optional" section so it's a bit less daunting